projects/admin/src/app/record/detail-view/circ-policy-detail-view/circ-policy-detail-view.component.html

```diff
@@ -22,148 +22,132 @@ <h1 class="mb-3">{{ data.name | translate }}</h1>
+            <ng-template #thenRequestBlock>
+              <i id="cipo-allow-request" class="fa fa-check  text-success" aria-hidden="true"></i>
+            </ng-template>
+            <ng-template #elseRequestBlock>
+              <i id="cipo-allow-request" class="fa fa-times text-danger" aria-hidden="true"></i>
+            </ng-template>
+          </dd>
+        </dl>
+        <!-- IS DEFAULT -->
+        <dl class="row mb-0">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Is default</dt>
+          <dd class="col-sm-7 col-md-8 mb-0">
+            <ng-container *ngIf="data.is_default; then thenDefaultBlock else elseDefaultBlock"></ng-container>
+            <ng-template #thenDefaultBlock>
+              <i id="cipo-is-default" class="fa fa-check  text-success" aria-hidden="true"></i>
+            </ng-template>
+            <ng-template #elseDefaultBlock>
+              <i id="cipo-is-default" class="fa fa-times text-danger" aria-hidden="true"></i>
+            </ng-template>
+          </dd>
+        </dl>
+      </div>
     </section>
     <!-- NOTIFICATION AND FEES -->
-    <section class="m-2 p-2" *ngIf="data.number_of_days_after_due_date &&
-        data.number_of_days_before_due_date  && data.reminder_fee_amount">
-      <h3 translate>Notifications and fees</h3>
-      <!-- DAYS AFTER DUE DATE -->
-      <dl class="row mb-0" *ngIf="data.number_of_days_after_due_date">
-        <dt class="col-sm-3 offset-sm-2 offset-md-0">
-          {{ 'Number of days after due date' | translate }}:
-        </dt>
-        <dd class="col-sm-7 col-md-8 mb-0">
-          {{ data.number_of_days_after_due_date }}
-        </dd>
-      </dl>
-      <!-- DAYS BEFORE DUE DATE -->
-      <dl class="row mb-0" *ngIf="data.number_of_days_before_due_date">
-        <dt class="col-sm-3 offset-sm-2 offset-md-0">
-          {{ 'Number of days before due date' | translate }}:
-        </dt>
-        <dd class="col-sm-7 col-md-8 mb-0">
-          {{ data.number_of_days_before_due_date }}
-        </dd>
-      </dl>
-      <!-- REMINDER FEE AMOUNT -->
-      <dl class="row mb-0" *ngIf="data.reminder_fee_amount">
-        <dt class="col-sm-3 offset-sm-2 offset-md-0">
-          {{ 'Reminder fee amount' | translate }}:
-        </dt>
-        <dd class="col-sm-7 col-md-8 mb-0">
-          {{ data.reminder_fee_amount }}
-        </dd>
-      </dl>
+    <section class="card mt-4" *ngIf="data.number_of_days_after_due_date ||
+        data.number_of_days_before_due_date || data.reminder_fee_amount">
+      <div class="card-header">
+        <b translate>Notifications and fees</b>
+      </div>
+      <div class="card-body">
+        <!-- DAYS AFTER DUE DATE -->
+        <dl class="row mb-0" *ngIf="data.number_of_days_after_due_date">
+          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Number of days after due date</dt>
```

Comment:
```suggestion
          <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Number of days after due date before sending a notification</dt>
```
Should be checked with @pronguen